README.md

```diff
@@ -5,325 +5,20 @@ time-series data. It is engineered up from PostgreSQL, providing automatic
 partitioning across time and space (partitioning key), as well as full
 SQL support.
 
-TimescaleDB is packaged as a PostgreSQL extension and set of scripts.
+TimescaleDB is packaged as a PostgreSQL extension and released under
+the Apache 2 open-source license.
 
-For a more detailed description of our architecture, [please read
-the technical
-paper](http://www.timescaledb.com/papers/timescaledb.pdf). Additionally,
-more documentation can be
-found [on our docs website](https://docs.timescaledb.com/).
+To install TimescaleDB, follow the instructions here:
 
-There are several ways to install TimescaleDB:
-1. Homebrew (for MacOS)
-1. `apt` (Ubuntu 16.04 & 17.04)
-1. Docker
-1. From source
+- [Installation Instructions](http://docs.timescale.com/getting-started/installation)
 
-## Installation
+Additional documentation can be found [on our docs
+website](http://docs.timescale.com/), including the following:
 
-_NOTE: Currently, upgrading to new versions requires a fresh install._
+- [Why use TimescaleDB?](http://docs.timescale.com/introduction)
+- [Migrating from PostgreSQL](http://docs.timescale.com/getting-started/setup/migrate-from-postgresql)
+- [Writing data](http://docs.timescale.com/api#insert)
+- [Querying and data analytics](http://docs.timescale.com/api#select)
+- [Tutorials and sample data](http://docs.timescale.com/tutorials)
+- [Contributing to TimescaleDB](https://github.com/timescale/timescaledb/blob/master/CONTRIBUTING.md)
```

Comment:
There was a request for the link.  Do you prefer pulling it out as a separate sentence?